packages/sdk/client-services/src/packlets/network/network-service.ts

```diff
@@ -2,17 +2,23 @@
 // Copyright 2022 DXOS.org
 //
 
+import { Stream } from '@dxos/codec-protobuf';
 import { NetworkManager } from '@dxos/network-manager';
-import { NetworkService, SetNetworkModeRequest } from '@dxos/protocols/proto/dxos/client/services';
+import { NetworkService, NetworkStatus, SetNetworkOptionsRequest } from '@dxos/protocols/proto/dxos/client/services';
 
 export class NetworkServiceImpl implements NetworkService {
   constructor(private readonly networkManager: NetworkManager) {}
 
-  async getNetworkMode() {
-    return { mode: this.networkManager.networkMode };
+  subscribeToNetworkStatus() {
```

Comment:
i think my comment was create...Subscription -- please check for consistency.

Reply:
I don't see such naming anywhere.